src/scripts/main.js

```diff
@@ -1 +1,73 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  const handler = () => {
+    resolve();
+    document.removeEventListener('click', resolve);
+  };
+
+  document.addEventListener('click', handler);
+
+  setTimeout(reject, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  const handler = () => {
+    resolve();
+    document.removeEventListener('click', resolve);
+    document.removeEventListener('contextmenu', resolve);
```

Comment:
You are trying to remove an event listener for 'contextmenu' event, but it was not added in this promise.